src/redux/types/profile.js

```diff
@@ -0,0 +1,2 @@
+export const SET_POST = 'SET-POST';
+export const ADD_POST = 'ADD-POST';
```

Comment:
Please use "_" underscore for the constants instead of "-" =)

It will be consistent with the names =)

Reply:
Done!